### PR TITLE
OnePointCutscene Cleanup

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/code/z_camera.c [hunk 1]
```diff
@@ -6240,6 +6246,8 @@ s32 Camera_Demo5(Camera* camera) {
 
     sDemo5PrevAction12Frame = camera->play->state.frames;
     Camera_ChangeSettingFlags(camera, CAM_SET_CS_C, (4 | 1));
+
+    // Continue to the general OnePoint cutscene update
     Camera_Unique9(camera);
```
Comment (fig02) [inline src/code/z_camera.c:6251]: im not sure if id describe this as "continue", its more like it wants to call this in addition to the stuff it just did for this frame
overall maybe just not worth commenting on (will be especially clear when we name these camera functions with better names)

Comment (engineer124) [reply src/code/z_camera.c:6251]: Yeah, I just wanted to highlight that this runs two "camera-update" functions, but as you mention, maybe that'll pop out once these camera-update functions are changed to better names. I can remove

**Review by fig02 — APPROVED** [review]

File: src/code/z_onepointdemo.c [hunk 2]
```diff
@@ -947,7 +948,7 @@ s32 OnePointCutscene_SetInfo(PlayState* play, s16 subCamId, s16 csId, Actor* act
 
         case 4220:
             csInfo->keyFrames = (player->actor.world.pos.z < -15.0f) ? D_80122C3C : D_80122C64;
-            csInfo->keyFrameCnt = 1;
+            csInfo->keyFrameCount = ARRAY_COUNT(D_80122C3C);
```
Comment (Dragorn421) [inline src/code/z_onepointdemo.c:951]: Questionable to me if array_count of one of the ternary branches is better than just "1" :shrug:

**Review by Dragorn421 — APPROVED** [review]

**Review by hensldm — APPROVED** [review]
LGTM :)

**Pull request merged**